lib/model/unreads.dart

```diff
@@ -223,6 +223,28 @@ class Unreads extends ChangeNotifier {
     }
   }
 
+  Set<int> get channelsWithUnreadMentions {
+    final channels = <int>{};
+    for (var messageId in mentions) {
+      final streamId = _streamIdsAndTopicsByMessageId[messageId]?.streamId;
+      if (streamId != null) {
+        channels.add(streamId);
+      }
+    }
+    return channels;
+  }
+
+  Set<int> get channelsWithUnmutedMentions {
+    final channels = <int>{};
+    for (var messageId in mentions) {
+      final info = _streamIdsAndTopicsByMessageId[messageId]!;
```

Comment:
nit:
```suggestion
      final (:streamId, :topic) = _streamIdsAndTopicsByMessageId[messageId]!;
```
This way we can avoid the `info` variable.